docs/source/containerization/kubernetes.mdx

```diff
@@ -1,246 +1,274 @@
 ---
 
 import { Link } from 'gatsby';
+import ElasticNotice from '../../shared/elastic-notice.mdx';
+import HelmShowOutput from '../../shared/helm-show-router-output.mdx';
+import K8SManualConfig from '../../shared/k8s-manual-config.mdx';
+import RouterCommonConfig from '../../shared/router-common-config.mdx';
 
-## Sample Kubernetes Configuration
+Learn how to deploy a self-hosted supergraph with GraphOS and the Apollo Router on Kubernetes: 
 
-> Note: The Apollo Router is made available under the Elastic License v2.0 (ELv2).  This applies to its source code and all distributions, including versions installed via Helm charts.  Read [our licensing page](https://www.apollographql.com/docs/resources/elastic-license-v2-faq/) for more details.
+* Get the Helm chart provided with each release of the router.
+* Divide a router configuration between one chart for common settings across all environments, and a separate chart for each specific environment.
+* Choose chart settings that best migrate from gateway to the router.
+* Install your Helm charts to deploy the router.
 
-### Helm
+<ElasticNotice />
 
-[Helm](https://helm.sh) is the package manager for kubernetes.
+## About the router Helm chart
 
-There is a complete [helm chart definition](https://github.com/apollographql/router/tree/v1.30.1/helm/chart/router) in the repo which illustrates how to use helm to deploy the router in kubernetes.
+[Helm](https://helm.sh) is a package manager for Kubernetes. You create a Helm _chart_ to describe the resources in a package then install the chart with Helm to deploy the package on Kubernetes (k8s).
 
-In both the following examples, we are using helm to install the router:
- - into namespace "router-deploy" (create namespace if it doesn't exist)
- - with helm install name "router-test"
- - with support for prometheus enabled
+Apollo provides an application Helm chart with each release of Apollo Router in GitHub. Each released Helm chart sets specific values for the router, including:
 
-#### Using helm chart from our Open Container Initiative (OCI) registry
+* Coprocessor
+* Header propagation
+* Max request sizes
+* Hot reload
+* CORS
+* Metrics and labels
 
-Starting with release 0.14.0, each time we release the router, we'll release
-our helm chart and store it in the same OCI registry in which we store our
-router docker images.
 
-You can use helm to install charts from an OCI registry as follows:
+## Get the router Helm chart
+
+The Helm chart for each release of the Apollo Router is available from the router's GitHub repository in `<repo>/helm/chart/router`. 
+
+> For example, the Helm chart of the latest release of the router is available [here](https://github.com/apollographql/router/tree/v1.31.1/helm/chart/router).
+
+Since the router v0.14.0, Apollo has also released a Helm chart for the router as an [Open Container Initiative (OCI)](https://helm.sh/docs/topics/registries/) image in the GitHub Container registry. 
```

Comment:
```suggestion
Since the router v0.14.0, Apollo has also released a Helm chart for the router as an [Open Container Initiative (OCI)](https://helm.sh/docs/topics/registries/) image in our GitHub Container registry. 
```